lib/solidus_affirm/factories.rb

```diff
@@ -1,6 +1,5 @@
-FactoryGirl.define do
-  # Define your Spree extensions Factories within this file to enable applications, and other extensions to use and override them.
-  #
-  # Example adding this to your spec_helper will load these Factories for use:
-  # require 'solidus_affirm/factories'
+require 'factory_girl'
+
+Dir["#{File.dirname(__FILE__)}/factories/**"].each do |f|
```

Comment:
We always manually require them since here they are in the `lib` directory so they are exposed outside of this extension as well for developers that like to use these factories in other extensions or in the host Rails app.